adaptived/src/effects/kill_cgroup_by_psi.c

```diff
@@ -66,6 +66,10 @@ int kill_cgroup_psi_init(struct adaptived_effect * const eff, struct json_object
 	if (ret)
 		goto error;
 
+	ret = adaptived_file_exists(cgroup_path_str);
```

Comment:
`kill_cgroup_by_psi` also allows wildcards.  See comment above in the PSI effect.